knowledge-base/grid-csv-export-change-field-delimiter.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Grid export to CSV - change the default field delimiter (comma)
```

Comment:
The word default is not used by the customer, so we can try shortening the title.

```suggestion
title: Grid CSV export - change the comma field delimiter
```